### DISCUSSION TITLE
How to untrack a file on android?

### DISCUSSION THREAD

**Discussion by lexst64** [Vinzent03/obsidian-git#662] [Q&A]
I've got an installed obsidian-git plugin on a android phone. When I commit all changes, workspace.json and workspace-mobile.json, which are listed in .gitignore, are committed too.

How to untrack them? I tried to delete them first and then to back them up but it didn't help since, when obsidian is opened, they are created again.

Unfortunately, couldn't find the answer in the web and here.

edit 1: typo

**Comment by Vinzent03** [comment]
There is currently no way to run `git rm --cached` which should be what you are searching for. For the moment you can try to delete the file from the repo on desktop or via GitHub website.
